tests/test_pep518.py

```diff
@@ -15,10 +15,9 @@
 @pytest.mark.parametrize(
     'build_arg', ['', '--wheel'], ids=['sdist_to_wheel', 'wheel_directly']
 )
-@pytest.mark.xfail(sys.platform.startswith('cygwin'), reason='ninja pip package not available on cygwin', strict=True)
```

Comment:
Why is this removed? The point of the test is to test that this breaks - that is, ninja's not available from PyPI on cygwin. So if this passes on Cygwin before #175 goes in, the test doesn't work. That's why it was a strict xfail.

Reply:
Okay, now I know why it's passing. That's okay, I think - then we can delete `ninja` from the wheelhouse manually for #175. The only think that lightly bothers me is the `check_call` change, then.

Reply:
I think that if we remove `ninja` from `meson-python`'s `install_require` list it will not be put in the wheelhouse anymore. But the documentation is not crystal clear on this. I'll have to check.

Reply:
I confirm that once `ninja` is removed from meson-python's `dependencies` it is not added to the wheelhouse anymore.